styleguide/components/21-atoms/link/_link.scss

```diff
@@ -6,37 +6,62 @@ a {
   text-decoration: underline;
   cursor: pointer;
 
+  &:hover {
+    @include theme('background-color', 'color-primary--lighten-4', 'link-background-hover');
+    @include theme('color', 'color-primary--darken-2', 'link-color-hover');
+  }
+
   &:focus {
     outline: thin dotted;
   }
 
-  //
-  // @TODO: Add hover state (waiting for design specifications.)
-  //
-  // &:hover {}
+  // Different button link types. These are links styled as a button.
+  &.button-link {
+    @include button;
 
-  // @TODO: Add external-link variant (waiting for design specifications.)
-  // &.external-link {
-  //   &::before {
-  //     @extend %icon-base-css;
-  //     @extend %icon-newtab;
+    display: inline-block;
+    line-height: inherit;
 
-  //     display: inline-block;
-  //     margin-right: .25rem;
-  //     text-decoration: none;
-  //   }
-  // }
+    i {
+      line-height: .7rem;
+    }
+
+    &-primary {
+      @extend %button-primary;
+    }
 
-  &.read-more-link {
-    @include icon('arrow-right', 'after');
+    &-secondary {
+      @extend %button-secondary;
+    }
+
+    &-dark {
+      @extend %button-dark;
+    }
+  }
 
-    line-height: 1rem;
+  // Standalone links (could be used for external links.)
+  &.stand-alone-link {
     text-decoration: none;
 
-    &::after {
-      position: relative;
-      top: .15rem;
-      margin-left: .5rem;
+    i {
+      line-height: normal;
```

Comment:
'Normal' depends on the user agent (1.2 in firefox).
Use a var or inherit from parent.